src/commands/music/queue.js

```diff
@@ -9,26 +10,146 @@ module.exports = class Queue extends Command {
     this.aliases = ['playlist']
     this.category = 'music'
 
+    this.subcommands = [
+      new QueueClear(client, this),
+      new QueueJump(client, this),
+      new QueueRemove(client, this),
+      new QueueShuffle(client, this)
+    ]
+
     this.requirements = new CommandRequirements(this, { guildOnly: true, guildPlaying: true })
   }
 
-  async run ({ t, author, channel, guild }) {
-    const guildPlayer = this.client.playerManager.get(guild.id)
+  async run ({ t, aliase, author, channel, guild, prefix }) {

```

Comment:
Hey, just a heads up: [The singular form of **aliases** is **alias**, not **aliases**](https://www.dictionary.com/browse/aliases).

Reply:
This is going to be fixed in another PR, I would have to change the CommandContext and every command that uses it.